### ISSUE TITLE
Please offer complete tutorial code for beginners.

### ISSUE THREAD

**Issue by MSKTHU** [NVIDIAGameWorks/nvrhi#5]
Could you please offer the complete code which could run even though a hello world project. I can hardly learn to use the API cause the only document I found is the uncomplete tutorial code.

**Comment by apanteleev** [comment]
Also, [this fork](https://github.com/SaeruHikari/nvrhi) looks like it has a complete running Vulkan app. I have not tested it though, use at your own risk.

**Comment by RobertBeckebans** [comment]
Is there a Discord for discussions about using NVRHI? I really like it. @stephenap07 started to write a backend for RBDOOM-3-BFG. We had several discussions about various next-gen wrapper for a good OpenGL replacement like The-Forge, BGFX, Dilligent Engine and so on. I recommended NVRHI and he made some very good progress in a short time. It's seems to be as easy as OpenGL and a dedicated Discord might help to spread its popularity.

**Comment by apanteleev** [comment]
No, there is not currently a Discord server for NVRHI. I don't think there is enough interest in it yet, but maybe creating one can help NVRHI gain popularity?..

If there is a server where you're discussing the RBDOOM3 work, I can join that and provide support if necessary.

**Comment by GoldenThumbs** [comment]
Has there been a Discord server or support forum created for this?

**Comment by apanteleev** [comment]
I recently learned about the Graphics Programming Discord server, and it looks like some people out there are aware of / using NVRHI. Feel free to join and maybe create a channel there. https://discord.com/invite/6mgNGk7

**Comment by lalishansh** [comment]
Hello @apanteleev, can you please provide complete end-to-end example with GLFW or SDL or Windows API (whichever you seem fit), all in a single source file (possibly Vulkan example) in plain old `int main()` style, as donut examples is daunting to look at while [tutorial.md] dosen't provide an end-to-end example and wasn't able get working application with it.
Thank you
